src/geocat/viz/util.py

```diff
@@ -179,6 +179,74 @@ def add_right_hand_axis(ax,
     return axRHS
 
 
+def add_height_from_pressure_axis(ax,
+                                  heights=None,
+                                  pressure_units='hPa',
+                                  ticklabelsize=12,
+                                  label='Height (km)',
+                                  labelpad=10,
+                                  axislabelsize=16):
+    """Utility function that adds a right-hand Height axis to the plot, derived
+    from the left-hand Pressure axis.
+
+    Args:
+
+        ax (:class:`matplotlib.axes._subplots.AxesSubplot` or :class:`cartopy.mpl.geoaxes.GeoAxesSubplot`):
+            Current axes to the current figure
+        heights (:class:`numpy.ndarray` or list):
+            An array of desired height values in km.
+        pressure_units (:class:`str`):
+            The Pint-compliant unit string associated with the Pressure values. Assume to be hPa.
+        ticklabelsize (:class:`int`):
+            Text font size of tick labels. A default value of 12 is used if nothing is set.
+        label (:class:`str`):
+            Text to use for the right hand side label.
+        labelpad (:class:`float`):
+            Spacing in points from the axes bounding box. A default value of 10 is used if nothing is set.
+        axislabelsize (:class:`int`):
+            Text font size for y-axes. A default value of 16 is used if nothing is set.
```

Comment:
Oh, it would also probably be a good idea to note that a lot of these are optional in the docstring